BitmovinAnalyticsCollector/Classes/Collector/util/Util.swift

```diff
@@ -83,6 +83,21 @@ class Util {
         }
         return codecs;
     }
+    
+    static func getStreamType(fromUrl: String) -> StreamType? {
```

Comment:
That's not the naming I mean, I didn't mean to combine the `from` and `url`, I wrote them separate, so that the method parameter name is `from` but the argument name is `url`. And without the `get` - you don't usually see it in Swift naming convention.